db/migrate/20240808083848_add_live_feedback_code_and_comments.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :course_assessment_live_feedbacks do |t|
       t.references :assessment, null: false, index: true, foreign_key: { to_table: :course_assessments }
       t.references :question, null: false, index: true, foreign_key: { to_table: :course_assessment_questions }
-      t.references :creator_course, null: false, index: true, foreign_key: { to_table: :course_users }
+      t.references :creator, null: false, index: true, foreign_key: { to_table: :users }
```

Comment:
put this change on the commit https://github.com/Coursemology/coursemology2/pull/7497/commits/e5eee60a30876a4725095c1d30dbe32c60d56385

again, one rule of thumb is to never edit the migration file.. if you think the new migration is necessary, create the new one

Reply:
done